src/sandstorm/grain.capnp

```diff
@@ -467,6 +588,40 @@ interface SessionContext {
   #     // pass event.data.token to your app's server and call SandstormApi.restore() with it
   #   }
   # }, false)
+
+  provide @4 (cap :Capability, requiredPermissions :PermissionSet,
+              descriptor :PowerboxDescriptor, displayInfo :PowerboxDisplayInfo);
+  # For sessions started with `newRequestSession()`, fulfills the original request. If only one
+  # capability was requested, the powerbox will close upon `provide()` being called. If multiple
+  # capabilities were requested, then the powerbox remains open and `provide()` may be called
+  # repeatedly.
+  #
+  # If the session was not started with `newRequestSession()`, this method is equivalent to
+  # `offer()`. This can be helpful when building a UI that can be used both embedded in the
+  # powerbox and stand-alone.
```

Comment:
I vaguely worry that there might be cases where, in the course of a powerbox request, the embedded app may want to `offer()` another capability that isn't meant to be a response to the request but is rather meant to start a new powerbox. Admittedly this seems like it would be crappy UX but I'm not sure it's safe to say it will never happen.

But maybe the answer here is that such an app should call `goToView()` (which I think I'm going to rename to `openView()`, BTW) to open a new tab and then make the new offer from there? Thoughts?